assets/runtime/env-defaults

```diff
@@ -24,6 +24,9 @@ fi
 GITLAB_SSH_HOST=${GITLAB_SSH_HOST:-$GITLAB_HOST}
 GITLAB_SSH_PORT=${GITLAB_SSH_PORT:-$GITLAB_SHELL_SSH_PORT} # for backwards compatibility
 GITLAB_SSH_PORT=${GITLAB_SSH_PORT:-22}
+SSH_HOST_KEYS_DIR=${SSH_HOST_KEYS_DIR:-${GITLAB_DATA_DIR}/ssh/}
+SSH_HOST_KEYS_DIR=${SSH_HOST_KEYS_DIR%/}/
```

Comment:
In line 28 it's a little bit confusing 😅 . Why do you remove first the trailing backslash and add it again.